### PR TITLE
Conseil 105 conseil should fetch individual accounts properly

### PR DESCRIPTION
**Pull request by ba205**
Commented out accounts

### CODE REVIEW

File: src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala [hunk 1]
```diff
@@ -16,7 +16,7 @@ trait Tables {
   /** DDL for all tables. Call .create to execute. */
```
Comment (vishakh) [inline src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala]: Why is this schema changing at all?

File: src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala [hunk 2]
```diff
@@ -139,6 +139,10 @@ class TezosNodeOperator(node: TezosRPCInterface) extends LazyLogging {
       getBlockHead(network).flatMap { blockHead =>
```
Comment (vishakh) [inline src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala]: Please remove extraneous comments.

File: src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala (hunk 1 above)
Comment (ba205) [reply src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala]: Oh, I reran GenSchema because my local wasn't up to date.

Comment (ba205) [reply src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala]: Also, this is just an untagged comment, so I'm sure it won't affect anything

Comment (vishakh) [reply src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala]: Looks like you dropped a primary key! That could hit performance a lot..

Comment (vishakh) [reply src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala]: Are you sure you ran the SQL from `conseil.sql`. We should not have any change in schema here.

**Comment by ba205** [comment]
Oh crap, I only saw this because it’s my personal email, my cryptonomic one isn’t linked to Github. I just left, would you want me to go back sometime this weekend to fix it? Is this blocking anyone? An alternative would be to pull the branch and commit the right Tables.scala file
-Basil
    On Friday, June 29, 2018, 3:42:10 PM EDT, Vishakh <notifications@github.com> wrote:  
 
 
@vishakh commented on this pull request.

In src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala:
> @@ -16,7 +16,7 @@ trait Tables {
   /** DDL for all tables. Call .create to execute. */

Looks like you dropped a primary key! That could hit performance a lot..

**Comment by vishakh** [comment]
Addresses #105

**Pull request merged**